### ISSUE TITLE
Howto use wildcard certs in all namespaces with auto renewal ?

### ISSUE THREAD

**Issue by barnybadzoo** [math-nao/certs#37]
With Certs i can use acme.sh for a lot of dns providers which are not being coverd with cert-manager, what is really great.
I did specify the env value for my provider in values.yaml, and was able to get for a single host an valid cert with an ingress like you describe at your site.

I would like to know how can i setup a wildcard domain cert for use in all namespaces ?

What i would like to achieve is this:
- Get a wildcard cert like this: 
`  acme.sh --issue --dns dns_doapi -d kube.example.com -d *.kube.example.com -d *.gitlab.kube.example.com -d *.ws.gitpod.kube.example.com -d *.pipe.kube.example.com`
- Import this cert as tls wildcard-domain-tls (can be done with kubectl)
- Distribute it to all namespaces as wildcard-domain-tls 
- with a renewal Job which renews the cert and updates the tls in every namespace

Or is this all already done by cert ?

**Comment by barnybadzoo** [comment]
i was able to solve the replication of the wildcard cert with kubed from appscode

**Comment by math-nao** [comment]
@barnybadzoo did you set `manageAllNamespaces`chart configuration to `true`?

**Comment by barnybadzoo** [comment]
yes i did use `manageAllNamespaces: true`. sorry for my late response...

Also i would like to know how ca i enable full debug output of acme.sh in logfiles.. i use debug: true but doesnt see them..

**Comment by EnterTheC** [comment]
New to kubernetes, question: Is there any chance to generate a wildcard certificate with certs, renew it using existing cron job and use this certificate with every configured ingress? If yes how would I get it working.
